packages/Search/cmake/ExplicitInstantiationSupport.cmake

```diff
@@ -94,4 +94,7 @@ TRIBITS_ETI_GENERATE_TYPEDEF_MACRO(DTK_ETI_TYPEDEFS "DTK_ETI_MANGLING_TYPEDEFS"
 # ${PACKAGE_NAME}_ETIHelperMacros.h.in
 CONFIGURE_FILE(
   ${DataTransferKit_SOURCE_DIR}/packages/Search/cmake/${PACKAGE_NAME}_ETIHelperMacros.h.in
+  ${DataTransferKit_BINARY_DIR}/packages/Search/src/details/${PACKAGE_NAME}_ETIHelperMacros.h)
```

Comment:
Why do we also need `details/...ETIHElperMacros.h`? This package has already generated one for us using the standard path.